slang.h

```diff
@@ -354,6 +369,8 @@ extern "C"
     typedef uint32_t    SlangUInt32;
     typedef intptr_t    SlangInt;
     typedef uintptr_t   SlangUInt;
+
+    typedef bool SlangBool;
```

Comment:
There are potential (and perhaps actual) speaking ABI issues around bool in com like interfaces, when using different compilers. This was perhaps more to keep in line with other use of slang types. If we wanted to avoid ABI issues we could use a int32_t, but it's somewhat clumsy in practice.